css/properties/justify-content.json

```diff
@@ -255,8 +255,7 @@
                   "version_added": "52"
                 },
                 "ie": {
-                  "version_added": false,
-                  "notes": "This value is recognized, but has no effect."
```

Comment:
In https://github.com/mdn/browser-compat-data/pull/3904 we agreed to leave this note in, but mark it as false (which is already is). So, please undo the note removal.

Reply:
Oh we did?  Note returning then!

Reply:
I think you've added it back at the wrong place?

Reply:
I think I did too, whoops -- I'm confused as to why it didn't show up in the diff, but fixed it now!